### ISSUE TITLE
Misleading documentation on tracing_subscriber::fmt filtering

### ISSUE THREAD

**Issue by djc** [tokio-rs/tracing#1357]
### Version

tracing-subscriber = 0.1.27

### Description

The documentation for `tracing_subscriber::fmt` states:

> The default collector installed by `init` enables you to filter events at runtime using environment variables (using the `EnvFilter`).

To me, this suggested that `.with_env_filter(EnvFilter::from_default_env())` was part of the `SubscriberBuilder::default()`, but I don't think it actually is.

Would be nice to clarify this.

**Comment by hawkw** [comment]
Yup, looks like the docs are incorrect. Thanks for pointing that out!
